dex-it-common/src/main/scala/com/wavesplatform/dex/it/api/dex/AsyncEnrichedDexApi.scala

```diff
@@ -199,6 +199,12 @@ class AsyncEnrichedDexApi(apiKey: String, host: => InetSocketAddress)(implicit e
       .followRedirects(true)
   }
 
+  override def orderBookInfo(amount: String, price: String): R[HttpOrderBookInfo] = mk {
```

Comment:
Let's get better names for arguments:
* amount -> amountAsset
* price -> priceAsset

Reply:
ok